### ISSUE TITLE
Revisit suppressed code analysis warnings in Microsoft.VisualStudio.AppDesigner

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#8183]
To port the source code for Microsoft.VisualStudio.AppDesigner and Microsoft.VisualStudio.Editors, we've needed to baseline some code analysis warnings so that we could check in:

``` VB
<Assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0015:Always consume the value returned by methods marked with PreserveSigAttribute", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.AppDesCommon.Utils.FocusFirstOrLastTabItem(System.IntPtr,System.Boolean)~System.Boolean")>
<Assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0015:Always consume the value returned by methods marked with PreserveSigAttribute", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.AppDesDesignerFramework.DeferrableWindowPaneProviderServiceBase.DesignerWindowPaneBase.OnUndoing(System.Object,System.EventArgs)")>
<Assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0015:Always consume the value returned by methods marked with PreserveSigAttribute", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.AppDesDesignerFramework.DeferrableWindowPaneProviderServiceBase.DesignerWindowPaneBase.OnUndone(System.Object,System.EventArgs)")>
<Assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0015:Always consume the value returned by methods marked with PreserveSigAttribute", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.PropertyPages.PropPageBase.Create(System.IntPtr)~System.IntPtr")>
<Assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0015:Always consume the value returned by methods marked with PreserveSigAttribute", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.PropertyPages.PropPageBase.Deactivate")>
<Assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0015:Always consume the value returned by methods marked with PreserveSigAttribute", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.PropertyPages.PropPageUserControlBase.PostValidation")>
<Assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0015:Always consume the value returned by methods marked with PreserveSigAttribute", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.PropPageDesigner.PropPageDesignerView.InitializeStateOfUICues")>
<Assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0015:Always consume the value returned by methods marked with PreserveSigAttribute", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.PropPageDesigner.PropPageDesignerView.PropertyPagePanel_GotFocus(System.Object,System.EventArgs)")>
<Assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0015:Always consume the value returned by methods marked with PreserveSigAttribute", Justification:="<Pending>", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.PropPageDesigner.PropPageDesignerView.UpdateWindowStyles(System.IntPtr)")>
```

We should revisit these, and investigate if we want to fix them.

**Comment by davkean** [comment]
Be sure to check GlobalSuppression.cs in each project.

**Comment by davkean** [comment]
This issue was moved to dotnet/roslyn-project-system#121
